renv.lock

```diff
@@ -65,6 +65,18 @@
       "Repository": "CRAN",
       "Hash": "f3ca785924863b0e4c8cb23b6a5c75a1"
     },
+    "admiral.test": {
+      "Package": "admiral.test",
+      "Version": "0.2.0",
+      "Source": "GitHub",
+      "RemoteType": "github",
+      "RemoteHost": "api.github.com",
+      "RemoteRepo": "admiral.test",
+      "RemoteUsername": "pharmaverse",
+      "RemoteRef": "devel",
+      "RemoteSha": "4f3bdd59a5a01bd0314a42e1d3ef32ca4d1745fe",
+      "Hash": "8378c5762f6be5a61854e880d6a8a223"
+    },
```

Comment:
@cicdguy , why is admiral.test added? I thought the upstream dependencies should not be in `renv.lock`.

Reply:
This was a test. Please ignore. I'll close this PR soon.